practices/managing-node-projects.md

```diff
@@ -135,3 +136,21 @@ Snyk not only provides alerts when a new vulnerability is detected in one of the
 [Using components with known vulnerabilities caused 24% of the top 50 breaches](https://snyk.io/blog/owasp-top-10-breaches/). Keeping dependencies up-to-date is therefore crucial to maintain our standards of security.
 
 Every node.js project must be added to the Springer Nature organisation in Snyk so it can be monitored for security vulnerabilities.
+
+## Handling dependencies from third parties
+
+A question that often comes up, should we commit the contents of the `node_modules` directory or any selected module. The answer on balance is that we don't recommend it. 
+
+Wherever possible we suggest instead:
```

Comment:
Semi-colon before lists, not colon.

Reply:
Also should suggest _one_ of the following alternatives, which is not clear.

Reply:
Actually it's a colon before lists not a semi-colon, semi-colon can be used a separator of items in a list to avoid ambiguity. Colon actually informs the reader that a list is coming.

Reply:
Ah yes my bad, apologies!

Reply:
Still a question with no question mark

Reply:
Please remove " or transpiled", the point is about compilation differences between host platforms, not transpiling source code.